README.md

```diff
@@ -11,80 +11,132 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        - [1.5.1. How to triage a release pipeline](#151-how-to-triage-a-release-pipeline)
+    - [1.6. How to setup Jenkins test environment](#16-how-to-setup-jenkins-test-environment)
 
-#### Guide to the Test Jobs at AdoptOpenJDK
+## 1.1. AQA Quick Start
 
-For nightly and release builds, there are test jobs running as part of the AdoptOpenJDK continuous delivery pipelines.  There is a [blog post and brief presentation](https://blog.adoptopenjdk.net/2017/12/testing-java-help-count-ways) that explains what testing we run and how they fit into the overall delivery pipeline.  As the world of testing at AdoptOpenJDK is evolving and improving quickly, some documentation may fall behind the march of progress.  Please let us know and help us keep it up-to-date, and ask questions at the [AdoptOpenJDK testing Slack channel](https://adoptopenjdk.slack.com/messages/C5219G28G)!
+To support development teams, AQA tests can be run locally.  These instructions assume you have set up your machine with the [test prereqs](https://github.com/eclipse/openj9/blob/master/test/docs/Prerequisites.md) and have built or downloaded the SDK you want to test.
 
-![CI pipeline view](doc/diagrams/ciPipeline.jpg)
+### 1.1.1. General steps to run locally
 
-#### Test 'Inventory'
+We run a diverse set of tests that use various underlying test frameworks.  In order to standardize them, we use TestKitGen ([TKG](https://github.com/AdoptOpenJDK/TKG)) which wraps them to provide a common look and feel.  These general steps are common to all test groups:
 
-The directory structure in this openjdk-tests repository is meant to reflect the different types of test we run (and pull from lots of other locations).  The diagrams below show the test make target for each of the types, along with in-plan, upcoming additions (denoted by dotted line grey boxes). The provided links jump to test jobs in Jenkins (ci.adoptopenjdk.net).
+1. State location of SDK you want to test (for example in /jdkHome)
+    - export TEST_JDK_HOME=/jdkHome 
+1. Clone openjdk-tests repo
+    - git clone https://github.com/AdoptOpenJDK/openjdk-tests.git (into /testLocation)
+    - cd openjdk-tests
+1. Fetch TKG and set appropriate environment variables 
+    - ./get.sh -t /testLocation/openjdk-tests
+    - cd TestConfig
+1. Export environment variables appropriate for tests you want to run  
+    - see [1.1.2. Examples](#112-examples)
+1. Generate applicable test targets (uses playlist.xml in test subdirectories)
+    - make -f run_configure.mk   
+1. Compile test material (uses build.xml files in test subdirectories)
+    - make compile 
+1. Execute test targets (by level/group/testCaseName defined in playlist.xml)
+    - make < _someTestTarget >    
 
-![overview of tests](doc/diagrams/overviewOfAdoptTests.svg)
+Note: Steps 4-7 can be repeated many times defining different environment variables to cumulatively incorporate different or more testing in the same workspace.
```

Comment:
I have set the following env variables:

```
export BUILD_LIST=functional // following 1.1.2 Examples
export TEST_JDK_HOME=~/src/openjdk-jdk/build/macosx-x86_64-server-release/jdk/ // folder containing bin/java
```
but my Step 4-7 failed. is the `TEST_JDK_HOME` incorrect?